tests/installation/disk_activation.pm

```diff
@@ -100,8 +100,8 @@ sub run {
             format_dasd;
         }
 
-        # format DASD if the variable is that, because we format it usually pre-installation
-        elsif (get_var('FORMAT_DASD_YAST')) {
+        # Test DASD formatting within YaST during installation
+        elsif (check_var('FORMAT_DASD_YAST', 1)) {
```

Comment:
So now you want to check for the three different values:

* FORMAT_DASD_YAST == 1 -> format in installer
* FORMAT_DASD_YAST defined but == 0 -> do not format
* FORMAT_DASD_YAST undefined -> format in bootloader_s390 from install shell

yes? I don't think this is a good idea and also it should not work as `if (get_var('FOO'))` should match if FOO is defined and is 1 already anyway. Could be confusing. Better we are explicit with different string values for the variable and we check the string, e.g.

* FORMAT_DASD=never equivalent to FORMAT_DASD undefined -> no format
* FORMAT_DASD=pre_install -> in bootloader_s390
* FORMAT_DASD=yast -> installation/disk_activation

Reply:
yeah, this came to my mind yesterday - and I already discussed it briefly with @rwx788 
the only drawback here is, since "pre-install" is and should be our default, we would need to set this on worker level or explicitly in the code as default and then overwrite it in special cases